collections/nemo_nlp/nemo_nlp/data/datasets/glue.py

```diff
@@ -0,0 +1,247 @@
+            segment_ids += [sequence_a_segment_id]
+
+        # Add special tokens to sequence_b
+        if tokens_b:
+            if bos_token:
+                tokens += [bos_token]
+                segment_ids += [sequence_b_segment_id]
+            tokens += tokens_b
+            segment_ids += [sequence_b_segment_id] * (len(tokens_b))
+            if eos_token:
+                tokens += [eos_token]
+                segment_ids += [sequence_b_segment_id]
+
+        # Add classification token - for BERT models
+        if cls_token:
+            if cls_token_at_end:
+                tokens += [cls_token]
+                segment_ids += [cls_token_segment_id]
+            else:
+                tokens = [cls_token] + tokens
+                segment_ids = [cls_token_segment_id] + segment_ids
+        input_ids = tokenizer.tokens_to_ids(tokens)
+
+        # The mask has 1 for real tokens and 0 for padding tokens. Only real
+        # tokens are attended to.
+        input_mask = [1 if mask_padding_with_zero else 0] * len(input_ids)
+
+        # Zero-pad up to the sequence length.
+        padding_length = max_seq_length - len(input_ids)
+        pad_token_id = tokenizer.tokens_to_ids([pad_token])[0]
+        if pad_on_left:
+            input_ids = ([pad_token_id] * padding_length) + input_ids
+            input_mask = ([0 if mask_padding_with_zero else 1] *
+                          padding_length) + input_mask
+            segment_ids = ([pad_token_segment_id] * padding_length) + \
+                segment_ids
+        else:
+            input_ids = input_ids + ([pad_token_id] * padding_length)
+            input_mask = input_mask + \
+                ([0 if mask_padding_with_zero else 1] * padding_length)
+            segment_ids = segment_ids + \
+                ([pad_token_segment_id] * padding_length)
+        if len(input_ids) != max_seq_length:
+            raise ValueError("input_ids must be of length max_seq_length")
+        if len(input_mask) != max_seq_length:
+            raise ValueError("input_mask must be of length max_seq_length")
+        if len(segment_ids) != max_seq_length:
+            raise ValueError("segment_ids must be of length max_seq_length")
+        if output_mode == "classification":
+            label_id = label_map[example.label]
+        elif output_mode == "regression":
+            label_id = np.float32(example.label)
+        else:
+            raise KeyError(output_mode)
+
+        if ex_index < 5:
+            logger.info("*** Example ***")
+            logger.info("guid: %s" % (example.guid))
+            logger.info(
+                "tokens: %s" % " ".join([str(x) for x in tokens]))
+            logger.info(
+                "input_ids: %s" % " ".join([str(x) for x in input_ids]))
+            logger.info(
+                "input_mask: %s" % " ".join([str(x) for x in input_mask]))
+            logger.info(
+                "segment_ids: %s" % " ".join([str(x) for x in segment_ids]))
```

Comment:
use map for this. e.g.:
"segment_ids: %s" % " ".join(list(map(str, segment_ids)))